tests/Test/TestCase.php

```diff
@@ -9,17 +9,21 @@
  * file that was distributed with this source code.
  */
 
-namespace Tests\Test;
+namespace Flarum\Tests\Test;
 
 use Mockery;
 use PHPUnit\Framework\TestCase as Test;
 
 abstract class TestCase extends Test
 {
+    use Concerns\CreatesForum, Concerns\MakesApiRequests;
+
     public function setUp()
     {
         Mockery::close();
```

Comment:
Wait a second, isn't `Mockery::close` supposed to be called on tear-down?

Reply:
Using mockery::close is part of the old tests. I'll check whether this is still needed.